src/StaticApi/Menus.php

```diff
@@ -24,11 +24,12 @@ public static function get_all_locations() {
 
 			foreach ( $menu_items as $menu_item ) {
 				if ( ! $menu_item->menu_item_parent ) {
-					$menus[ $location ][] = [
+					$menus = [
 						'title' => $menu_item->title,
 						'link' => str_replace( site_url(), '', $menu_item->url ),
 						'items' => self::get_sub_menu_items( $menu_items, $menu_item->ID ),
 					];
+					$menus[ $location ][] = apply_filters( "menu_$location", $menus );
```

Comment:
Does this work when you have multiple menu locations? It looks like you're overwriting the $menus var on each loop in the line above.